pkgs/by-name/ja/jazz2/package.nix

```diff
@@ -0,0 +1,56 @@
+{ cmake
+, fetchFromGitHub
+, glfw
+, jazz2-content
+, lib
+, libopenmpt
+, libvorbis
+, openal
+, SDL2
+, stdenv
+, testers
+, zlib
+, graphicsLibrary ? "GLFW"
+}:
+
+assert lib.assertOneOf "graphicsLibrary" graphicsLibrary [ "SDL2" "GLFW" ];
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jazz2";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "deathkiller";
+    repo = "jazz2-native";
+    rev = finalAttrs.version;
+    hash = "sha256-1psMeuMV8GjS+uNlgtCvKpHgV9XW+vjviQTHBPjA4Lc=";
+  };
+
+  patches = [ ./nocontent.patch ];
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ libopenmpt libvorbis openal zlib ]
+  ++ lib.optionals (graphicsLibrary == "GLFW") [ glfw ]
+  ++ lib.optionals (graphicsLibrary == "SDL2") [ SDL2 ];
+
+  cmakeFlags = [
+    "-DLIBOPENMPT_INCLUDE_DIR=${lib.getDev libopenmpt}/include/libopenmpt"
+    "-DNCINE_DOWNLOAD_DEPENDENCIES=OFF"
+    "-DNCINE_OVERRIDE_CONTENT_PATH=${jazz2-content}"
+  ] ++ lib.optionals (graphicsLibrary == "GLFW") [
+    "-DGLFW_INCLUDE_DIR=${glfw}/include/GLFW"
+  ];
+
+  passthru.tests.version = testers.testVersion {
+    package = finalAttrs.finalPackage;
+    command = "${finalAttrs.finalPackage.meta.mainProgram} --version";
```

Comment:
Not necessary as that's the default. Only need to specify `command` if the program has a version flag other than `--version`, needs more setup to work in the test environment, or normally returns a non-zero status code when printing the version. Checked locally, and default works fine.

```suggestion
```

Reply:
done, thanks